@stellar/design-system-website/src/styles.scss

```diff
@@ -1,18 +1,42 @@
-.Layout__content {
-  .Layout__inset {
-    & > * {
-      margin-bottom: 2rem;
-    }
+.Section {
+  margin-bottom: 2rem;
+
+  ul {
+    margin-bottom: 1.5rem;
+  }
+}
+
+.ComponentContainer {
+  border-bottom: 1px solid var(--color-border-main);
+  padding-bottom: 1.5rem;
+}
+
+.ComponentExamplesContainer {
+  margin-bottom: 1.5rem;
+
+  & > * {
+    margin-bottom: 1rem;
+    max-width: 30rem;
+  }
+
+  .IconButton {
+    margin-bottom: 1rem;
   }
 }
 
+.TableContainer {
+  position: relative;
+  overflow: auto;
+  margin-top: 2rem;
+}
+
 table {
   width: 100%;
   border-collapse: collapse;
   font-size: 1rem;
   line-height: 1.5rem;
   color: var(--color-text);
-  margin-top: 2rem;
+  min-width: 900px;
```

Comment:
i'm assuming this px value is based on the media query width. May be helpful to save as a variable?

Reply:
Yeah, this is based on the table `min-width`, I'll make it a variable. 👍 Thanks for pointing it out!